packages/lib/services/commands/ToolbarButtonUtils.ts

```diff
@@ -33,6 +33,24 @@ export default class ToolbarButtonUtils {
 		return this.service_;
 	}
 
+	// Editor commands will focus the editor after they're executed
+	private isEditorCommand(commandName: string) {
+		return (commandName.indexOf('editor.') === 0 ||
+				// These commands are grandfathered in, but in the future
+				// all editor commands should start with "editor."
+				commandName === 'textBold' ||
+				commandName === 'textItalic' ||
+				commandName === 'textCode' ||
+				commandName === 'attachFile' ||
+				commandName === 'textNumberedList' ||
+				commandName === 'textBulletedList' ||
+				commandName === 'textCheckbox' ||
+				commandName === 'textHeading' ||
+				commandName === 'textHorizontalRule' ||
+				commandName === 'insertDateTime'
```

Comment:
It looks like a few of them are missing, like textLink, textCopy, etc. shouldn't they be added too? For info, the full list is in editorCommandDeclaration.ts

Reply:
Thanks, that was an oversight. I actually copied these commands out of editorCommandDeclaration.ts but missed the top few by mistake.